driver/driver.py

```diff
@@ -0,0 +1,130 @@
+import argparse
+import os
+import yaml
+import shutil
+import logging
+import tempfile
+import subprocess
+from pathlib import Path
+from wxflow import Logger, Jinja
+import jinja2
+from datetime import datetime
+from dateutil.parser import parse as parse_datetime
+
+
+def generate_eva_config(template_path: str, output_path: str, context: dict):
+    """
+    Render a Jinja template to create an EVA configuration file.
+
+    Args:
+        template_path (str): Path to the Jinja template file.
+        output_path (str): Path where the rendered config file will be saved.
+        context (dict): Dictionary containing the data used for rendering the template.
+
+    Returns:
+        str: Path to the rendered EVA configuration file.
+    """
+    jinja_render = Jinja(template_path_or_string=template_path, data=context)
+    jinja_render.save(output_file=output_path)
+    return output_path
+
+
+def driver(monitor_dict: dict):
+    """
+    Main driver function for observation monitoring workflow.
+
+    This function:
+    - Creates a timestamped runtime directory
+    - Copies NetCDF files matching the observation type to that directory
+    - Generates a Jinja2-based EVA configuration file
+    - Runs the EVA command using the config
+    - (Optional) Copies the results to an output directory
+
+    Args:
+        monitor_dict (dict): Dictionary containing configuration values
+    """
+
+    # Add logger
+    logger = Logger('Observation Monitoring')
+
+    logger.info('Starting Observation Monitoring')
+
+    # Validate required directories
+    experiment_dir = Path(monitor_dict["experiment_dir"])
+    if not experiment_dir.exists():
+        raise FileNotFoundError(f"Experiment directory not found: {experiment_dir}")
+
+    runtime_root = Path(monitor_dict.get("runtime_dir", "./runtime"))
+    outdir = Path(monitor_dict.get("outdir", "./outdir"))
+    template_path = monitor_dict["template_path"]
+
+    if not Path(template_path).exists():
+        raise FileNotFoundError(f"Template path not found: {template_path}")
+
+    # Create runtime directory
+    timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
```

Comment:
we should get PDY and cyc from the environment to get the cycle time and not the exact current UTC time, will make it easier to find what we are looking for